cratedb_toolkit/io/dynamodb/backlog.md

```diff
@@ -0,0 +1,9 @@
+# DynamoDB Backlog
+
+- Pagination / Batch Getting.
+  https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/programming-with-python.html#programming-with-python-pagination
+
+- Use `batch_get_item`.
+  https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/dynamodb/client/batch_get_item.html
+
+- Scan by query instead of full.
```

Comment:
@wierdvanderhaar: With respect to alternative implementations, using batched reading from DynamoDB is probably way to go when processing large amounts of data?

Reply:
Yes, we should use the batched method. Let's use a default batch size of 100 but give the option to use different batch sizes.

Reply:
Hi. I added the remnant items to the backlog.

- GH-231

Do you agree to merge and release it first, following the "first make it work, then make it fast|beautiful|robust" paradigm, in order to get it out as quickly as possible?